change/@ni-nimble-components-292ee238-87d4-40bc-b76c-941992c1f31e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Let's add link to the new spec from the table column and formatted text specs, similar to what Mert's doing in #1566.

Reply:
Done, but note that only the formatted text spec was altered to point to the new spec in Mert's changes. I've followed suit.